apis/discovery/v1alpha1/foreigncluster_types.go

```diff
@@ -83,6 +93,11 @@ type ForeignClusterSpec struct {
 	// +kubebuilder:default="Auto"
 	// +kubebuilder:validation:Optional
 	IncomingPeeringEnabled PeeringEnabledType `json:"incomingPeeringEnabled"`
+	// Indicates if Liqo has to handle the network interconnection with the remote cluster.
+	// +kubebuilder:validation:Enum="No";"Yes"
+	// +kubebuilder:default="Yes"
+	// +kubebuilder:validation:Optional
+	NetworkingEnabled NetworkingEnabledType `json:"networkingEnabled,omitempty"`
```

Comment:
Removing OmitEmpty flag if the field is not set then the empty string is set as value. Obviously the creation would fail since the `""` is not a valid value.

Reply:
You are right, just ignore my comment